modules/sfp_sublist3r.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:         sfp_sublist3r
+# Purpose:      SpiderFoot plug-in for subdomain enumeration using
+#               api.sublist3r.com
+#
+# Author:      TheTechromancer
+#
+# Created:     05/21/2021
+# Copyright:   (c) Steve Micallef 2021
+# Licence:     GPL
+# -------------------------------------------------------------------------------
+
+import json
+
+from spiderfoot import SpiderFootEvent, SpiderFootPlugin
+
+
+class sfp_sublist3r(SpiderFootPlugin):
+
+    meta = {
+        "name": "Sublist3r PassiveDNS",
+        "summary": "Passive subdomain enumeration using Sublist3r's API",
+        "useCases": ["Investigate", "Footprint", "Passive"],
+        "categories": ["Passive DNS"],
+        "dataSource": {
+            "website": "https://api.sublist3r.com",
+            "model": "FREE_NOAUTH_UNLIMITED",
+            "description": "This is the API queried by the Sublist3r tool.",
+        }
+    }
+
+    # Default options
+    opts = {}
+
+    # Option descriptions
+    optdescs = {}
+
+    def setup(self, sfc, userOpts=dict()):
+        self.sf = sfc
+        self.sf.debug("Setting up sfp_sublist3r")
+        self.state = self.tempStorage()
+        self.state.update({
+            "events": [],
+        })
+        self.__dataSource__ = "DNS"
```

Comment:
This shouldn't be overridden since the data source is really Sublist3r, not DNS directly.